tests/FakeItEasy.Tests/Core/DefaultFakeManagerAccessorTests.cs

```diff
@@ -99,7 +99,7 @@ public void Should_fail_when_getting_manager_from_object_where_a_fake_manager_ha
             // Assert
             exception.Should()
                 .BeAnExceptionOfType<ArgumentException>()
-                .WithMessage("The specified object is not recognized as a fake object.");
+                .WithMessage($"Object '{proxy.ToString()}' of type '{proxy.GetType().FullNameCSharp()}' is not recognized as a fake object.");

```

Comment:
Ah, I see. You can do something like this instead:

             exception.Should()
                 .BeAnExceptionOfType<ArgumentException>()
                 .Which.Message.Should().Match("Object 'Faked FakeItEasy.Creation.ITaggable' of type 'Castle.Proxies.ObjectProxy*' is not recognized as a fake object.");